datasets/disaster_response_messages/README.md

```diff
@@ -0,0 +1,165 @@
+---
+{"task_categories": [intent-classification, sentiment-analysis],"multilinguality": [multilingual], "languages": [multiple], "language_creators": [Unknown], "annotations_creators": [Unknown], "source_datasets": [Original], "size_categories": ["10K<n<100K"], "licenses": [Unknown]}
```

Comment:
can you use the YAML instead of JSON here ? (even though the tagger app allows to export in JSON as well)